proposals/lip-Introduce_cross-chain_update_mechanism.md

```diff
@@ -128,8 +133,11 @@ Create a cross-chain update transaction for a given height `h1`:
     In this case, the `messageWitness` will be empty.
 *   Compute the inclusion proof for the outbox root of the sidechain account into the mainchain state root. 
     This proof is then used to compute `inboxUpdate.outboxRootWitness`.
-*   If the `validatorsHash` property of the certificate is different from the `validatorsHash` derived from the `validators` property of the mainchain account on the sidechain, compute the required update between both sets and include it in `validatorsUpdate`. 
-    How to obtain this update is detailed in the [Appendix](#appendix).
+*   If the `validatorsHash` property of the certificate is different from the `validatorsHash` derived from the `activeValidators` and `certificateThreshold` properties of the mainchain account on the sidechain: 
+   *   Compute the required active validator update between the `activeValidators` stored in the chain account 
```

Comment:
I find this sentence a bit confusing, maybe it could be simplified to smtg like:

"Compute the update of the `activeValidators` stored in the chain account by following the logic of `getActiveValidatorsDiff` as detailed in the [Appendix](#appendix). "

Reply:
yes, it was not the best wording. Rephrased.